frontend/src/pages/ModelTester/ChartComponenet2/ChartComponent2.jsx

```diff
@@ -11,21 +11,6 @@ const ChartComponent2 = forwardRef(({ chartData }, ref) => {
     useEffect(() => {
         if (!chartData) return;
 
-        const labels = chartData.labels || []; // Expect labels to be passed in chartData as an array of "Model 1", "Model 2", ...
-
-        const datasets = [
-            {
-                label: "Generated Data", // Label for the dataset
-                data: chartData.datasets?.[0]?.data || [], // Data values corresponding to the models
-                backgroundColor: labels.map((_, index) => `rgba(${index * 40}, ${100 + index * 40}, ${150 - index * 30}, 0.6)`), // Dynamic color for each bar
```

Comment:
why was this part of code removed

Reply:
This was used initially as placeholder data but now that we are fetching data from the API, it is no longer needed